components/membership/membership-rest-impl/src/main/kotlin/net/corda/membership/impl/rest/v1/MemberLookupRestResourceImpl.kt

```diff
@@ -45,7 +52,8 @@ class MemberLookupRestResourceImpl @Activate constructor(
             organizationUnit: String?,
             locality: String?,
             state: String?,
-            country: String?
+            country: String?,
+            statuses: List<String>,
```

Comment:
```suggestion
            statuses: Set<String>,
```
as order does not matter and we do not need to have duplicating strings in the `List`.

Reply:
Addressed in [e599ed7](https://github.com/corda/corda-runtime-os/pull/3362/commits/e599ed71dd0d8e3ad6a2ea7190cdad4ebb66ddfd)